### DISCUSSION TITLE
Blazor in .NET 8 - New "Interactive Server" render mode

### DISCUSSION THREAD

**Discussion by danielchalmers** [Live-Charts/LiveCharts2#1355] [Q&A]
It's been stated that the library won't work on Blazor Server due to the static rendering (#606 #1274), but will it work using the new [hybrid render modes](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/render-modes?view=aspnetcore-8.0) introduced in .NET 8?

Ran into this issue on a Blazor Server hybrid app that runs locally as a replacement to a WPF app, and also had the issue trying to add the charts to a MAUI Blazor app which seems to run as Server too. Would be super valuable to be able to use it with them.

Thanks all.

**Comment by beto-rodriguez** [comment]
I will see what this new render mode can do, I am not completely sure of the capabilities of it.

About Maui/Blazor, as far as I know, Maui uses a web view to display the Blazor UI, it is basically just a browser, I think that the current version of LiveCharts should work there, but I have not tried that yet.

**Reply by danielchalmers** [reply]
Thank you sir. When I tried the library on Maui Blazor I got an error that I believe said something about it not being available for server-side apps.
